rasa/nlu/components.py

```diff
@@ -484,6 +517,6 @@ def create_component(
             return component
         except MissingArgumentError as e:  # pragma: no cover
             raise Exception(
-                "Failed to create component `{}`. "
-                "{}".format(component_config["name"], e)
+                f"Failed to create component `{component_config['name']}`. "
```

Comment:
Do we have a standard for single quotes vs backticks for this stuff?

Reply:
I think it is quotes for names of things and backticks for code

Reply:
yes, should be, will update it